### DISCUSSION TITLE
tutorials/export/exporting_for_windows

### DISCUSSION THREAD

**Discussion by giscus** [godotengine/godot-docs-user-notes#216] [User-contributed notes]
# tutorials/export/exporting_for_windows

The simplest way to distribute a game for PC is to copy the executable ( godot.exe), compress the folder and send it to someone else. However, this is often not desired. Godot offers a more elegant...

https://docs.godotengine.org/en/stable/tutorials/export/exporting_for_windows.html

**Comment by percellas** [comment]
the link for "information on creating one can be found here." links to a microsoft page stating that makecert.exe is deprecated. That microsoft page recommends using this method instead: https://learn.microsoft.com/en-us/windows/msix/package/create-certificate-package-signing

**Reply by Calinou** [reply]
Pull request opened:

- https://github.com/godotengine/godot-docs/pull/10128

**Comment by Damglador** [comment]
I think it should also include info about rcedit, and how it's configured on Linux

**Reply by Calinou** [reply]
Pull request opened:

- https://github.com/godotengine/godot-docs/pull/10965
